crates/wit-component/src/encoding/wit/v1.rs

```diff
@@ -170,6 +170,10 @@ impl InterfaceEncoder<'_> {
         let iface = &self.resolve.interfaces[interface];
         let mut type_order = IndexSet::new();
         for (_, id) in iface.types.iter() {
+            let ty = &self.resolve.types[*id];
+            if let TypeOwner::Interface(iface_id) = ty.owner {
+                self.interface = Some(iface_id);
+            }
```

Comment:
Is this still necessary? I ask because the v1 encoding is sort of dead at this point and I should probably work to remove it entirely, so in theory no more updates should be necessary.